Description: Remove a hack for Sun that causes FTBFS with glibc >= 2.27
Author: Adrian Bunk <bunk@debian.org>

--- uudeview-0.5.20.orig/tcl/uutcl.c
+++ uudeview-0.5.20/tcl/uutcl.c
@@ -48,14 +48,6 @@
 #include <tcl.h>
 #endif
 
-/*
- * The following variable is a special hack that is needed in order for
- * Sun shared libraries to be used for Tcl.
- */
-
-extern int matherr();
-int *tclDummyMathPtr = (int *) matherr;
-
 #include <uudeview.h>
 #include <uuint.h>
 #include <fptools.h>
